packages/wrangler/src/versions/upload.ts

```diff
@@ -323,6 +333,13 @@ export const versionsUploadCommand = createCommand({
 			);
 		}
 
+		const previewAlias =
+			args.previewAlias !== undefined
+				? args.previewAlias
+				: getCIGeneratePreviewAlias()
+					? generatePreviewAlias(name)
+					: undefined;
```

Comment:
NIT this is the same as:

```suggestion
			args.previewAlias ??
				getCIGeneratePreviewAlias()
					? generatePreviewAlias(name)
					: undefined;
```

Reply:
Ack, took me a hot second to figure out, I think this was running into operator precedence weirdness. Wrapping the parens makes this work 👍